format/index.md

```diff
@@ -873,15 +880,22 @@ for each relationship name can be specified:
 GET /articles/1?include=comments.author
 ```
 
-> Note: A request for `comments.author` should not automatically also
-include `comments` in the response. This can happen if the client already
-has the `comments` locally, and now wants to fetch the associated authors
-without fetching the comments again.
+> Note: Because compound documents require full linkage, intermediate
+resources in a multi-part path must be returned along with the leaf nodes.
+For example, a response to a request for `comments.author` should
+automatically include `comments` as well as the `author` of each of those
+`comments`.
+
+> Note: A server may choose to expose a deeply nested relationship such as
```

Comment:
Is this sentence consistent with the requirement above that "each relationship name **MUST** be identical to the key in the `"relationships"` member of its parent resource object"? That would seem to imply that, in a request for `?include=comments.author`, the relationship names are `"comments"` and `"author"`, but not "comment-authors". Also, allowing a key like `"comment-authors"` creates an ugly read-write asymmetry.